### ISSUE TITLE
[Bug]: Videos cannot be fetched to display on any channel feed even when "Fetch feeds from RSS" is enabled

### ISSUE THREAD

**Issue by ReaFinley7** [FreeTubeApp/FreeTube#9223]
### Guidelines

- [x] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [x] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads). **The only official downloads are from the [https://freetubeapp.io](https://freetubeapp.io/#download) website, the [FreeTubeApp/FreeTube GitHub repository](https://github.com/FreeTubeApp/FreeTube/releases) and the [io.freetubeapp.FreeTube flatpak](https://flathub.org/en/apps/io.freetubeapp.FreeTube) from Flathub.**
- [x] I have [searched the issue tracker for **open and closed** issues](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+sort%3Arelevance-desc) that are similar to the bug report I want to file, without success.
- [x] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [x] This issue contains only one bug.
- [x] I have read and agree to follow the [rules](https://docs.freetubeapp.io/community/rules/).

### Describe the bug

1. Type any channel in the search bar. Click on the channel.
2. Channel dashboard displays no videos. An error pops up that says: TypeError: Cannot read properties of undefined (reading 'metadata_parts'). This problem is only for videos; not shorts, playlists, or posts or about section. However, when searching the title for a video in the "search channel" bar, the video then shows up and I can watch it. 
3. Someone said a workaround is to enable "Fetch Feeds from RSS" under the subscriptions tab in settings. This does not work.

I just downloaded Freetube on windows today.

### Expected Behavior

I expected to see all of the channel's latest videos when I click on the channel after searching it up. 

### Issue Labels

accessibility issue, content not loading

### FreeTube Version

v0.24.0

### Operating System Version

Windows 11 Pro version 25H2

### Installation Method

.exe

### Primary API used

Local API

### Upload screenshots or videos

[Screenshot 2026-05-29 215816.png](https://github.com/user-attachments/assets/0e88dc7f-d5b2-4c35-a4da-f94829c033cc)

### Nightly Build

- [ ] I have encountered this bug in the latest **official** [nightly build](https://docs.freetubeapp.io/development/nightly-builds), do not check this box if you are using an unofficial or custom build.

**Comment by SkyTheGreatOne** [comment]
I search for `.+` on a channel to show the videos. (Though it won't be sorted, it's better than nothing.) Sometimes reloading and doing it again is required as on occasion it will show seemingly random videos.
